### ISSUE TITLE
[1] Common names mapping file

### ISSUE THREAD

**Issue by hyanwong** [EOL/tramea#297]
Off the back of https://github.com/EOL/tramea/issues/162, I wonder how difficult it would be to produce a single database dump for common names, with columns:

VernacularName, lang, preferred, EoLPageID

Where there could be multiple entries for each page, one for each name. Perhaps a list of sources for each name might be useful too, although this might be more problematic for e.g. user-inputted names.

**Comment by JRice** [comment]
Indeed, this would not be too hard; I'd just need a little time to get it done.

I've estimated the ticket—we can try and fit it into an iteration, as possible...
